### PR TITLE
[14.0][MIG] base_import_match: Migration to 14

### PR DESCRIPTION
**Pull request by chusamo**
[functional video](https://www.loom.com/share/fbf5f13287f24aa79a0b4699d3d27c34)

[Contact 1](https://ibb.co/YyB2YpF)
[Contact 2](https://ibb.co/D7CbW5K)

### CODE REVIEW

**Review by anasuarez1 — APPROVED** [review]
LGTM!

File: base_import_match/models/base.py [hunk 1]
```diff
@@ -35,6 +35,7 @@ def load(self, fields, data):
                 if xmlid:
                     # Skip rows with ID, they do not need all this
                     row["id"] = xmlid
+                    newdata.append(tuple(row[f] for f in clean_fields))
                     continue
```
Comment (chusamo) [inline base_import_match/models/base.py:39]: It is added to the newdata list otherwise it won't affect any record even if it is found by external_id

**Review by xavierjimenez — APPROVED** [review]
Code review LGTM

**Comment by chusamo** [comment]
@OCA/community-maintainers  Could you check this PR please?

**Comment by chusamo** [comment]
@pedrobaeza @sebastienbeau Could you check this PR please?

**Pull request merged**